app/components/wallet/WalletAdd.js

```diff
@@ -104,56 +99,40 @@ export default class WalletAdd extends Component<Props> {
     }
 
     return (
-      <div className={componentClasses}>
-        <div className={buttonsContainerClasses}>
-          {!classicTheme && (
-            <div className={styles.title}>{title}</div>
-          )}
-
-          <Button
-            className="primary"
-            label={intl.formatMessage(messages.useLedgerDescription)}
-            onMouseUp={onLedger}
-            skin={ButtonSkin}
-          />
-          <Button
-            className="primary"
-            label={intl.formatMessage(messages.useTrezorDescription)}
-            onMouseUp={onTrezor}
-            skin={ButtonSkin}
-          />
-          <Button
-            className={createWalletButtonClasses}
-            label={intl.formatMessage(messages.createDescription)}
-            onMouseUp={onCreate}
-            skin={ButtonSkin}
-          />
-          <Button
-            className={restoreWalletButtonClasses}
-            label={intl.formatMessage(messages.restoreDescription)}
-            onMouseUp={onRestore}
-            skin={ButtonSkin}
-          />
-          <Button
-            className="primary restorePaperWalletButton"
-            label={intl.formatMessage(messages.restorePaperDescription)}
-            onMouseUp={onPaperRestore}
-            skin={ButtonSkin}
-          />
-          {activeNotification ? (
-            <div className={styles.notification}>
-              <FormattedHTMLMessage
-                {...messages[activeNotification]}
-                values={{ maxWalletsCount: MAX_ADA_WALLETS_COUNT }}
-              />
+      <div className={styles.hero}>
+        <div className={styles.heroInner}>
+          <div className={styles.heroLeft}>
+            <SvgInline svg={logo} className={styles.heroLogo} />
+            <h2 className={styles.title}>{title}</h2>
+            <h3 className={styles.subTitle}>{intl.formatMessage(messages.subTitle)}</h3>
+          </div>
+          <HorizontalFlexContainer>
+            {/* eslint-disable */}
```

Comment:
Why did eslint have to be disabled here?

Reply:
fixed with button wrapper for cards. there was an error about accessibility.